From 22a6e5b3f51783fc10e3fc7327a42bdcb92912e8 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Wed, 4 Oct 2017 17:59:20 +0200
Subject: [PATCH 05/14] sepolicy: Add sysfs labels for devices using 'soc.0'

* Newer QCOM kernels seem to be using 'soc', this
  allows older kernels to have proper labels on sysfs.

Change-Id: I60a732efcd369c0cb95c10b15b661b14f41b5aca
---
 legacy-common/file_contexts | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 legacy-common/file_contexts

diff --git a/legacy-common/file_contexts b/legacy-common/file_contexts
new file mode 100644
index 0000000..e17e697
--- /dev/null
+++ b/legacy-common/file_contexts
@@ -0,0 +1,10 @@
+# Battery
+/sys/devices(/platform)?/soc\.0/qpnp-linear-charger-[a-z0-9]+/power_supply/battery(/.*)?   u:object_r:sysfs_battery_supply:s0
+/sys/devices(/platform)?/soc\.0/qpnp-vm-bms-[a-z0-9]+/power_supply/bms(/.*)?   u:object_r:sysfs_battery_supply:s0
+
+# Graphics
+/sys/devices(/platform)?/soc\.0/[a-z0-9]+.qcom,mdss_mdp/qcom,mdss_fb_primary\.[0-9]+/leds/lcd-backlight(/.*)?   u:object_r:sysfs_graphics:s0
+/sys/devices(/platform)?/soc\.0/[a-z0-9]+.qcom,mdss_mdp/caps   u:object_r:sysfs_graphics:s0
+
+# Subsys restart
+/sys/devices(/platform)?/soc\.0/[a-z0-9\.:]+,[a-z0-9\-]+/subsys[0-9]+/name    u:object_r:sysfs_ssr:s0
-- 
2.7.4

